internal/ls/completions.go

```diff
@@ -31,12 +31,16 @@ func (l *LanguageService) ProvideCompletion(
 	preferences *UserPreferences,
 ) (*lsproto.CompletionList, error) {
 	program, file := l.getProgramAndFile(documentURI)
+	var triggerCharacter *string
```

Comment:
Refactored to get rid of nil reference crashes when accessing client capabilities and trigger character.